lego/apps/events/permissions.py

```diff
@@ -10,6 +16,38 @@ class EventPermissionHandler(PermissionHandler):
 
     perms_without_object = [CREATE, "administrate"]
 
+    def event_type_keyword_permissions(self, event_type, perm):
+        """
+        Get the keyword permission string required for a permission for a specific event type
+
+        :return: The keyword permission the user needs
+        """
+        from lego.apps.events.models import Event
+
+        return self.keyword_permission(Event, perm) + "{event_type}/".format(
+            event_type=event_type
+        )
+
+    def has_event_type_level_permission(self, user, request, perm):
+        """
+        Check if a user has the required event_type
+        permissions for a certain action.
+
+        Return `True` if the user has permission to perform the request
+        """
+        if request is None:
+            return True
```

Comment:
Is this ever used for something? My initial thought is that it should be `False`?

Reply:
I think this might be left over from when before I moved the permission check to the view. I'll check it out and remove/change it to `False`. Thanks :+1: